ApiCalls.js

```diff
@@ -26,6 +26,21 @@ export const addFlyToAPI = async (flyData) => {
   }
 };
 
+export const addUpdatedFly = async (flyData) => {
+  try {
+    const response = await fetch(`https://my-fly-box-api.herokuapp.com/api/v1/flies/${flyData.id}`, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8'
+      },
+      body: JSON.stringify(flyData.attributes),
+    });
```

Comment:
Nice work on this PATCH request.  I wasn't 100% sure how this would be set up, but you were able to create the correct setup.  This will be useful when we use PATCH request in the Fish Form.